tests/test_data_api_impl.py

```diff
@@ -513,3 +513,10 @@ async def test_flow_csv_generation(populated_db):
         "destination_code",
         "value",
     ]
+
+@pytest.mark.skip("Development test only - needs a bucket to run correctly")
+@pytest.mark.asyncio
+async def test_generate_signed_dqs_url():
+    url = await data_api_impl.generate_signed_dqs_url()
+    print(url)
+    assert "Mobility" in url.url
```

Comment:
I've used this test as a dev convenience, but the function it calls is so straightfoward and to mock it properly looks like excessive effort. If we change the test suite to have an 'online' run condition, it'd be worth it but I don't think we need that just for this one endpoint where most of the issues we'll have will be with bucket configuration.